app/services/CartServiceImpl.java

```diff
@@ -135,7 +135,7 @@ private String getFrequencyString(final Cart cart) {
     public F.Promise<Integer> getFrequency(final String cartId) {
         requireNonNull(cartId);
         final F.Promise<CustomObject<JsonNode>> customObjectPromise =
-                playJavaSphereClient().execute(CustomObjectByKeyGet.of(PactasKeys.FREQUENCY, cartId));
+                playJavaSphereClient().execute(CustomObjectByKeyGet.of(PactasKeys.FREQUENCY, cartId, JsonNode.class));
```

Comment:
actually there is a method `CustomObjectByKeyGet.ofJsonNode`

http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/customobjects/queries/CustomObjectByKeyGet.html#ofJsonNode-java.lang.String-java.lang.String-